Common/AnnoParser.hs

```diff
@@ -237,7 +238,8 @@ floatingAnno ps = literal2idsAnno ps Float_anno
 
 prefixAnno :: Range -> GenParser Char st Annotation
 prefixAnno ps = do
-    prefixes <- many $ do
+    prefixes <- many $ skipMany (commentLine >> endOfLine) >> do
```

Comment:
I'm not sure if you really need `endOfLine`. I suspect that `spaces` also skips `crlf`. If you really need to check for the end of a line, then `newline` (from `Text.ParserCombinators.Parsec.Char`) should do, too.